tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php

```diff
@@ -0,0 +1,1973 @@
+<?php
```

Comment:
We will need to make sure we add this test to the new test framework

Reply:
checked +1